extras/mmu/mmu.py

```diff
@@ -3903,7 +3918,8 @@ def _unload_gate(self, homing_max=None):
                 msg = "did not clear the encoder after moving %.1fmm" % homing_max
 
         else: # Using mmu_gate or mmu_post_gate sensor
-            actual,homed,_,_ = self.trace_filament_move("Reverse homing to gate sensor", -homing_max, motor="gear", homing_move=-1, endstop_name=self._get_gate_endstop_name())
+            endstop_name = self._get_gate_endstop_name()
+            actual,homed,_,_ = self.trace_filament_move("Reverse homing to %s sensor" % endstop_name, -homing_max, motor="gear", homing_move=-1, endstop_name=endstop_name)
```

Comment:
Everything here and beyond is just including sensor names in the logs/errors for easier debugging.